account/client.go

```diff
@@ -18,76 +21,83 @@ func New(params *stripe.AccountParams) (*stripe.Account, error) {
 	return getC().New(params)
 }
 
+// New creates a new account.
```

Comment:
I understand adding comments to a library but I'm not a fan of really short/terse comments like this. I don't know if it's a Go thing, but I wanted to flag just once.
Also, does it do like javadoc where you comment each parameter and returned values and errors?

Reply:
Yeah, I kind of agree, but it's a bit of a compromise — we could go with something much more long-winded, but it has a higher chance of falling out of date more quickly. I went with this instead, and admittedly it's not great documentation, but it will have the advantage of at least forcing documentation for all exported types/functions later on if we're able to successfully activate Golint.

I'm actually up for trying an alternative if you have a good suggestion. One approach is that we could try to copy what's in the reference documentation, but again, I feel like it might be in slight danger of becoming outdated:

> To charge a credit card or other payment source, you create a Charge object. If your API key is in test mode, the supplied payment source (e.g., card) won't actually be charged, although everything else will occur as if in live mode. (Stripe assumes that the charge would have completed successfully).

Regarding commenting on each parameter and returned value: no this isn't really a thing in Go. Instead, important parameters and return values are expected to be mentioned in the normal documentation string. You can find some examples of this in the Godoc for the standard library:

https://godoc.org/net/http#Redirect

Reply:
> I'm actually up for trying an alternative if you have a good suggestion. One approach is that we could try to copy what's in the reference documentation, but again, I feel like it might be in slight danger of becoming outdated

I fully agree. That's why I mostly removed all of those when I did the merge. I feel like the methods are self explanatory and devs should look at our official API ref instead though I don't know how Go devs work. For me it's the same as the ongoing PRs to tag things in stripe-php.

But I think your explanation makes sense and I'm happy to move forward with this.

Reply:
Thanks that sounds good. I think if we eventually go to auto-generation, all of this could possibly be less big of a deal because we'll be able to easily change the source and distribute updates out to everywhere (after updating all those sources to be a little more language-agnostic of course).

One more possibility: there were direct links before in a couple of these to the associated action in the API reference documentation like:

https://stripe.com/docs/api/go#create_charge

I could put those back in, and at least that might the docstrings a little more helpful. What do you think?

Reply:
I would punt. More than once we ended up "guessing" the URLs while adding something gated that did not have docs already and it's been not maintained in general.

Reply:
True — that has definitely been a problem in the past. Okay, thanks! I'm going to merge this and we can see how the rest of it goes.